testing/execution.ts

```diff
@@ -460,6 +460,8 @@ async function executeFormulaOrSyncWithRawParamsInVM<T extends FormulaSpecificat
       // to an autocomplete metadata formula), because at execution time it gets passed as a serialized
       // JSON string anyway which is already parsed by the compiled pack definition.
       params = rawParams as ParamValues<ParamDefs>;
+      // Default the search string (first arg) to an empty string.
+      params[0] ??= '';
```

Comment:
Can you make this same change at line 513 too? We unfortunately have slightly different execution paths for with-vm and without, the reason why we can't reuse I can't recall atm.

Reply:
Done!